.github/workflows/rubocop.yml

```diff
@@ -104,8 +104,8 @@ jobs:
         rubocop: [master]
         internal_investigation: [null]
         include:
-          - { rubocop: master, ruby: "3.0", os: ubuntu }
-          - { rubocop: "v1.69.2", ruby: "3.4", os: ubuntu }
+          - { rubocop: master, ruby: "3.4", os: ubuntu }
+          - { rubocop: "v1.61.0", ruby: "2.7", os: ubuntu }
```

Comment:
Can you leave comments explaining the intent of the tests for each of these two matrices to clarify the purpose of the versions?

Reply:
👍 I added 60143df02c5c6a0e552eb1343765ed598572c806

Thanks